edit/edit.js

```diff
@@ -1530,6 +1573,13 @@ function updateLintReportIfEnabled(...args) {
 }
 
 function save() {
+  if (!editor) {
+    return _save();
+  }
+  editor.save();
+}
+
+function _save() {
```

Comment:
This is a meaningless underscore, it doesn't tell anything. Either rename the function or rework the code to remove the need to use this function.